src/backend/linux_raw/mount/types.rs

```diff
@@ -330,3 +332,13 @@ bitflags! {
 
 #[repr(transparent)]
 pub(crate) struct MountFlagsArg(pub(crate) c::c_uint);
+
+#[repr(C)]
+#[derive(Debug, Copy, Clone)]
+#[allow(missing_docs)]
+pub struct MountAttr<'a> {
+    pub attr_set: MountAttrFlags,
```

Comment:
According to Linux's documentation, `attr_set` is a `__u64`, while `MountAttrFlags` is currently a `c_uint`. Could you add a test testing that the layout matches, following one of the "layouts" tests in the tree?